packages/nektar/package.py

```diff
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import llnl.util.filesystem as fs
```

Comment:
Just to confirm, this always come bundled with spack and we don't need any extra python config?

Reply:
Yes. It is used extensively within Spack itself.